tests/performance/locust_files/metamodeling/workflow.py

```diff
@@ -0,0 +1,159 @@
+from datetime import timedelta
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from typing import Final
+from uuid import UUID
+
+from locust import HttpUser, task
+from pydantic import Field
+from pydantic_settings import BaseSettings
+from requests.auth import HTTPBasicAuth
+from tenacity import (
+    Retrying,
+    retry_if_exception_type,
+    stop_after_delay,
+    wait_exponential,
+)
+from urllib3 import PoolManager, Retry
+
+_MAX_WAIT_SECONDS: Final[int] = 60
+
+
+# Perform the following setup in order to run this load test:
+# 1. Copy .env-devel to .env in this directory and add your osparc keys to .env.
+# 2. Construct a study **template** according to study_template.png. passer.py is the file next to this file.
+# 3. Setup the locust settings in the .env file (see https://docs.locust.io/en/stable/configuration.html#all-available-configuration-options)
+# run 'make test target=metamodeling/workflow.py' in your terminal and watch the magic happen 🤩
+
+
+class UserSettings(BaseSettings):
+    osparc_api_key: str = Field(default=...)
```

Comment:
Q: I wonder whether these settings will capture uppercase `osparc_api_key` , `osparc_api_secret` from `.env` file

Reply:
That works. I use them like that in the e2e client tests too.